src/edu/csus/ecs/pc2/core/Utilities.java

```diff
@@ -78,12 +79,17 @@ public final class Utilities {
     public static final String ISO_8601_TIMEDATE_FORMAT_WITH_MS = "yyyy-MM-dd'T'HH:mm:ss.SSSX";
 
     /**
-     * CCS directory where data files are stored (under problem short name).
+     * CLICS directory where judge's (secret) data file are stored
      * 
      * @see #getSecretDataPath(String, Problem)
      * @see #getSecretDataPath(String, String)
      */
-    public static final String SECRET_DATA_DIR = "data" + File.separator + "secret";
+    public static final String SECRET_DATA_DIR = "data" + File.separator + ExportYAML.SECRET_DIRECTORY_NAME;
+    
+    /**
+     * CLICS directory where judge's sample data file are stored
+     */
+    public static final String SAMPLE_DATA_DIR = "data" + File.separator + ExportYAML.SAMPLE_DIRECTORY_NAME;
```

Comment:
The same comment which I made above (in regard to having two different definitions of the secret directory name) applies equally here:  having different constants `SAMPLE_DATA_DIR` and `SAMPLE_DIRECTORY_NAME` is confusing; I think one of the names should be changed (and I think `SAMPLE_DIRECTORY_NAME` doesn't belong in `ExportYAML`).